File: 29 - Invert the Case of Characters in a String/solution.py
def invert_case(s):
    result = ""

    for char in s:
        if char.islower():
            result += char.upper()
        elif char.isupper():
            result += char.lower()
        else:
            result += char

    return result

# Test Cases    
print(invert_case("Hello, World!"))  # hELLO, wORLD!

File: 29 - Invert the Case of Characters in a String/solution.js
function invertCase(str) {
  let result = "";

  for (let char of str) {
    if (char === char.toLowerCase()) {
      result += char.toUpperCase();
    } else if (char === char.toUpperCase()) {
      result += char.toLowerCase();
    } else {
      result += char;
    }
  }

  return result;
}

// Test Cases
console.log(invertCase("Hello, World!")); // hELLO, wORLD!
